assets/main.css

```diff
@@ -14,7 +14,7 @@ dl, dd, ol, ul, figure {
  * Basic styling
  */
 body {
-	font: 400 16px/1.5 'Source Sans Pro', sans-serif;}
+	font: 400 16px/1.5 'Source Sans Pro', sans-serif;
```

Comment:
@shaqsingh -- want to make sure to point out this issue, are the colour and background-colour properties still going to be intended?